sdk/lib/atl/atlwin.h

```diff
@@ -1800,6 +1800,15 @@ public:
             return TRUE;                                                                        \
     }
 
+#define COMMAND_CODE_HANDLER(code, func)                                                            \
+    if (uMsg == WM_COMMAND && code == HIWORD(wParam))                                                \
+    {                                                                                            \
+        bHandled = TRUE;                                                                        \
+        lResult = func(HIWORD(wParam), LOWORD(wParam), (HWND)lParam, bHandled);                    \
```

Comment:
I'm not sure I understand how the bHandled parameter could be changed by the function "func"...

Reply:
like this: https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/mspaint/dialogs.h;hb=2367206744dc728b054560e81a1461f035edf699#l26

Reply:
Ohoh, subtle :D